content/blog/2016-03-03-custom-alertmanager-templates.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Custom Alertmanager Templates
+created_at: 2016-03-03
+kind: article
+author_name: Fabian Reinartz
+---
+
+The Alertmanager handles alerts sent by Prometheus servers and sends
+notifications about them to different receivers based on their labels.
+
+A receiver can be one of many different integrations such as PagerDuty, Slack,
+email, or a custom integration via the generic webhook interface (for example [JIRA](https://github.com/fabxc/jiralerts)).
+
+## Templates
+
+The messages sent to receivers are constructed via templates.
+Alertmanager comes with default templates but also allows defining custom
+ones.
+
+In this blog post, we will walk through a simple customization of Slack
+notifications.
+
+We use this simple Alertmanager configuartion that sends all alerts to Slack:
+
+```yaml
+global:
+  slack_api_url: '<slack_webhook_url>'
+
+route:
+  receiver: 'slack-notifications'
+  # All alerts in a notification share these labels.
+  group_by: [alertname, datacenter, app]
+  
+receivers:
+- name: 'slack-notifications'
+  slack_configs:
+  - channel: '#alerts'
+```
+
+By default, a Slack message sent by Alertmanager looks like this:
+
+![](http://imgur.com/Rormurm.png)
```

Comment:
Agreed. 

@juliusv the DreamHack ones are now moved to (or taken from) the DreamHack flickr account. Can/should we host those ourselves too?